### PR TITLE
remove version in wallet

### LINKED ISSUE #1193: Remove version check logic
https://github.com/vegaprotocol/vegawallet/issues/474#event-6099169827

- Remove version check logic
- Remove version displays

### PR DESCRIPTION
**Pull request by dexturr**
Closes #1193

### CODE REVIEW

**Review by edd — APPROVED** [review]

**Comment by dexturr** [comment]
Thanks @edd, realized this actually needed a lot more 🙈 

@mattrussell36 would be good to get your eyes on this one as well as I haven't done much in the wallet place

File: src/components/vega-wallet/vega-wallet-container.tsx [hunk 1]
```diff
@@ -23,7 +23,8 @@ export const VegaWalletFormContainer = () => {
     const run = async () => {
       if (!url) return;
       vegaWalletService.url = url;
-      const [version] = await vegaWalletService.getVersion();
+      const [version] = await vegaWalletService.getKeys();
+      console.log(version);
```
Comment (mattrussell36) [inline src/components/vega-wallet/vega-wallet-container.tsx:27]: Stray console.log here and variable name should change

```suggestion
      const [err] = await vegWalletService.getKeys();
```

File: src/lib/vega-wallet/vega-wallet-service.ts [hunk 2]
```diff
@@ -1,5 +1,3 @@
-import semver from "semver";
```
Comment (mattrussell36) [inline src/lib/vega-wallet/vega-wallet-service.ts:1]: I think this is the only place we use this package so it can be removed from package.json

**Review by mattrussell36 — CHANGES_REQUESTED** [review]
Couple fixes required here.

**Pull request merged**